packages/core/README.md

```diff
@@ -272,3 +273,15 @@ To enable logging, call the following method specifying the path of the log dire
 ```typescript
 enableLogging({ logPath: '/path/to/my/log/directory' });
 ```
+
+### isTokenValid
+
+```typescript
+  isTokenValid(): Promise<boolean>;
+```
+
+Once you connected to the SDK with your key and a user ID, you can check the validity of the generated token by calling:
```

Comment:
@cyrilbo Once you **are** connected to the SDK with your **api** key and a user ID